src/config/config.ts

```diff
@@ -134,12 +134,17 @@ export class Configuration {
   };
 
   payIn = {
-    minPayIn: {
+    min: {
       DeFiChain: {
         DFI: 0.0001,
         DUSD: 1,
       },
     },
+    forward: {
+      phrase: process.env.FORWARD_PHRASE?.split(','),
```

Comment:
Add `?? []`

Reply:
have taken it from kyc.phrase - should we add a default there too? 🤔

Reply:
Yes, good idea 👍